Misc/reputation-cve.json

```diff
@@ -7,11 +7,11 @@
   "commitMessage": "",
   "shouldPublish": false,
   "shouldCommit": false,
-  "regex": "",
+  "regex": "CVE-\\d{4}-\\d{4,7}",
   "details": "CVE",
   "prevDetails": "CVE",
-  "reputationScriptName": "",
-  "reputationCommand": "",
+  "reputationScriptName": "cveReputation",
```

Comment:
I'm not sure whether this is necessary or relevant - I saw it on the old indicator so I added it. @bakatzir do you know?

Reply:
this automation is not a formatting script as unescapeips, what it does is run the `cve-search` command. which is a hack. we should implement in this integration a `cve` command instead.

Reply:
If you wish to only merge this indicator. you can leave this for now, and open an issue to implement a `cve` comamnd instead in all the relevant integrations(which include `CVE Search` and `Kenna v2` AFAIK).

Reply:
There's already an issue for that.